exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,13 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function greeting(name) {
+  return "hello " + name.toUpperCase();
+}
+
```

Comment:
your code is perfect , but instead of writing console.log for each mentor name you can use for loop to write all mentors name .
that will be easier and simpler.

```suggestion
for(let i =0; i<mentors.length , i++){

```

Reply:
@Ezgi3406 yeaah, sure good suggestion!